supabase/templates/invite.html

```diff
@@ -0,0 +1,7 @@
+<html>
+  <body>
+    <h2>You have been invited</h2>
+    <p>You have been invited to create a user on {{ .SiteURL }}. Follow this link to accept the invite:</p>
+    <p><a href="{{ .ConfirmationURL }}/auth-callback">Accept the invite</a></p>
```

Comment:
After investigation, it appears this is the wrong solution. We shouldn't need to change the email template for the `/auth-callback`. Instead, we should modify the `authProvider` so that it specify the `/auth-callback` route as the `redirectTo` parameter of the supabase method calls that need it.

However, this change might break existing applications that followed the current documentation, so this is a breaking change.